src/OrleansRuntime/Placement/IPlacementContext.cs

```diff
@@ -68,9 +72,9 @@ public static string GetGrainTypeName(this IPlacementContext @this, GrainId grai
             return @this.GetGrainTypeName(grainId.GetTypeCode(), genericArguments);
         }
 
-        public static void GetGrainTypeInfo(this IPlacementContext @this, GrainId grainId, out string grainClass, out PlacementStrategy placement, string genericArguments = null)
+        public static void GetGrainTypeInfo(this IPlacementContext @this, GrainId grainId, out string grainClass, out PlacementStrategy placement, out MultiClusterRegistrationStrategy activationStrategy, string genericArguments = null)
```

Comment:
(Maybe once this is merged) we should create a `GrainTypeInfo` struct to hold the `out` parameters